### ISSUE TITLE
New line in odo delete

### ISSUE THREAD

**Issue by mik-dass** [redhat-developer/odo#398]
```[mdas@localhost odo]$ ./odo delete wildly1
Are you sure you want to delete wildly1 from app? [y/N] y
buildconfig "wildly1" deleted
imagestream "wildly1" deleted
deploymentconfig "wildly1" deleted
service "wildly1" deleted

Default component set to: nodejsnew[mdas@localhost odo]$ 
```

**Comment by surajnarwade** [comment]
Fixed in #390

**Comment by mik-dass** [comment]
OK :+1:
